src/main/java/com/faforever/api/error/ErrorResult.java

```diff
@@ -0,0 +1,39 @@
+package com.faforever.api.error;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.RequiredArgsConstructor;
+
+import java.util.Map;
+
+@Data
+@RequiredArgsConstructor
+@AllArgsConstructor
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class ErrorResult {
+  @JsonProperty("status")
+  private final String httpStatusCode;
+  @JsonProperty("code")
+  private String appCode;
+  private final String title;
+  private final String detail;
+  private Meta meta;
+
+  @Data
+  @RequiredArgsConstructor
+  @AllArgsConstructor
+  @JsonInclude(JsonInclude.Include.NON_NULL)
+  public static class Meta {
+    private String[] args;
```

Comment:
An argument may also be an integer, so `args` can be `Object[]`. Then you don't need the `getArgsAsStrings().